project/SoqlStdlib.scala

```diff
@@ -3,6 +3,11 @@ import Keys._
 
 object SoqlStdlib {
   lazy val settings: Seq[Setting[_]] = BuildSettings.projectSettings() ++ Seq(
-    name := "soql-stdlib"
+    name := "soql-stdlib",
+    libraryDependencies ++= Seq(
+      "com.rojoma" %% "rojoma-json-v3" % "3.13.0",
+      "com.rojoma" %% "simple-arm-v2" % "2.3.1"
```

Comment:
That seems like a good idea - I was trying to avoid adding additional projects (prior to having these two dependencies) - I can go ahead and break it out 😄